src/twisted/internet/unix.py

```diff
@@ -501,7 +501,9 @@ def _bindSocket(self):
         self.fileno = self.socket.fileno
 
     def write(self, datagram, address):
-        """Write a datagram."""
+        """Write a datagram.
+           raises socket.error EAGAIN
```

Comment:
Twisted uses epydoc syntax for comments, so this should use `@raise` or `@raises`.  See: http://epydoc.sourceforge.net/manual-fields.html or other places in the Twisted code.

Reply:
Done.